basicdataset/src/test/resources/mlrepo/dataset/nlp/ai/djl/basicdataset/amazon_reviews/metadata.json

```diff
@@ -0,0 +1,42 @@
+{
+  "metadataVersion": "0.2",
+  "resourceType": "dataset",
+  "application": "nlp",
+  "groupId": "ai.djl.basicdataset",
+  "artifactId": "amazon_reviews",
+  "name": "amazon_reviews",
+  "description": "A sentiment analysis dataset of Amazon reviews",
+  "website": "https://s3.amazonaws.com/amazon-reviews-pds/readme.html",
+  "artifacts": [
+    {
+      "version": "1.0",
+      "snapshot": false,
+      "name": "amazon_reviews_us_Digital_Software",
+      "properties": {
+        "dataset": "us_Digital_Software"
+      },
+      "files": {
+        "amazon_reviews": {
+          "uri": "https://s3.amazonaws.com/amazon-reviews-pds/tsv/amazon_reviews_us_Digital_Software_v1_00.tsv.gz",
+          "sha1Hash": "b8390100b92579ed814eede4112514417e339902",
+          "size": 18997559
+        }
+      }
+    },
+    {
+      "version": "1.0",
```

Comment:
Amazon reviews is a popular collection of datasets, we should provide them.
I only added two here for now.